app/api/chat/[id]/route.ts

```diff
@@ -26,21 +26,21 @@ export async function GET(
   );
 
   if (!session || !session.userId) {
-    return new Response("Unauthorized", { status: 401 });
```

Comment:
there is not big advantage of this change, they actually moving to support native methods like Response

So I don't think we need to change although it's okay.